exposee-ui/src/components/Mainpage/Main.jsx

```diff
@@ -74,13 +112,18 @@ function Main() {
         >
           {videos.map((video, index) => (
             <div
-              className={`video-container ${
-                index === currentIndex ? "active" : ""
-              }`}
+              className={`video-container ${index === currentIndex ? "active" : ""}`}
               key={video.id}
             >
               <VideoPlayer playbackId={video.mux_playback_id} />
               <p>{video.title}</p>
+              {/* Add the gift button */}
```

Comment:
this is already done right? i see Gift button in the video so.

Reply:
it helps my reviewer what is coming next (ie. adding the gift button to the videos )